### PR TITLE
refactor: use single implementation of sheet

### PR DESCRIPTION
**Pull request by satya164**
This will allow us to use the same API in browser and node, and will be useful for things like custom properties.

### CODE REVIEW

File: src/sheet.js [hunk 1]
```diff
@@ -3,53 +3,22 @@
...
-      dump() {
-        const result = cssText || '';
-        cssText = null;
-        cache = {};
-        return result;
-      },
-    };
-  }
+  const browser =
+    typeof window === 'object' && window != null && window.document != null;
```
Comment (thymikee) [inline src/sheet.js]: since it's boolean, `isBrowser` maybe?

File: src/sheet.js [hunk 2]
```diff
@@ -3,53 +3,22 @@
...
+  // eslint-disable-next-line global-require
+  const document = browser ? window.document : require('./document').default;
```
Comment (thymikee) [inline src/sheet.js]: No such file?

Comment (satya164) [reply src/sheet.js]: What do you mean?

Comment (thymikee) [reply src/sheet.js]: Sorry, didn't spot this

screen shot 2017-09-11 at 16 52 14

**Pull request merged**